src/app/components/notification.ts

```diff
@@ -8,9 +8,13 @@ import {
 import {Store} from '@ngrx/store';
 import {NotificationActions} from '../actions/actions';
 import {Notification} from '../interfaces/notification.interface';
-// why does this freak out!!!
-import styles from './notificationStyles.styl';
+
 import 'rxjs/add/operator/do';
+// why does this freak out!!!
+// import styles from './notificationStyles.styl'; // you also have a typo
+
+// you need to use require because this is a weboack feature
+var styles = require('./notificationStyle.styl');
```

Comment:
`require` must be used as it's a webpack feature to require other file types. TypeScript is working to allow other types to be typed